src/app/components/ClearAllButton/ClearAllButton.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+import styled from 'styled-components'
+
+type ClearAllButtonProps = {
+  children: ReactNode
+  onClick: () => void
+}
+
+export default function ClearAllButton({
+  children,
+  onClick,
+}: ClearAllButtonProps): JSX.Element {
+  return <Button onClick={onClick}>{children}</Button>
+}
+
+const Button = styled.button`
+  padding: 0.8em;
+  justify-self: center;
+  background-color: var(--color-choral);
+  border: none;
+  border-radius: 3em;
+  color: var(--color-background-light);
+  grid-area: clear;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
```

Comment:
maybe you can also make a variable out of the box-shadow? Plus, maybe you can give it rem or em instead of px. But, that's up to you :)

Reply:
absolutely! thx!